formats/finder_email_signup/publisher/details.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+   "required": [
+     "details"
+   ],
+  "properties": {
+     "details": {
+       "$ref": "#/definitions/details"
+     }
+  },
+  "definitions": {
+    "details": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "beta": {
+          "type": "boolean"
+        },
+        "email_signup_choice": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "$ref": "#/definitions/email_signup_choice"
```

Comment:
This doesn't need to be a ref to a separate definition, since it's only used in one place. Move the definition here.